src/test/compile-fail/import-from-missing.rs

```diff
@@ -16,3 +16,4 @@ mod spam {
 }
 
 fn main() { ham(); eggs(); }
+//~^ ERROR unresolved name `eggs`
```

Comment:
Yeah, I was thinking the same thing. I think we do want to avoid this and I plan to file an issue once this PR lands